### PR TITLE
Turn HTML sanitizer into a plugin

### PR DESCRIPTION
**Pull request by ohrely**
## References

https://github.com/jupyterlab/jupyterlab/issues/7384

## Code changes

- Creates a sanitizer plugin in `apputils-extension`.
- Provides new sanitizer plugin to rendermime registry.

## User-facing changes

No visual changes anticipated.

## Backwards-incompatible changes

No backwards-incompatible changes anticipated.

### CODE REVIEW

**Review by blink1073 — APPROVED** [review]
Thanks!

**Comment by blink1073** [comment]
@jasongrout I'll let you merge in case you have any follow up concerns.

File: packages/apputils/src/tokens.ts [hunk 1]
```diff
@@ -137,3 +137,50 @@ export namespace IThemeManager {
     unload(): Promise<void>;
   }
 }
+
+/* tslint:disable */
```
Comment (jasongrout) [inline packages/apputils/src/tokens.ts:141]: I think we don't need to disable tslint here. Can you remove this line?

```suggestion
```

Comment (jasongrout) [inline packages/apputils/src/tokens.ts:141]: I think we don't need to disable tslint here. Can you remove this line?

```suggestion
```

File: packages/apputils/src/tokens.ts [hunk 2]
```diff
@@ -137,3 +137,50 @@ export namespace IThemeManager {
...
+/**
+ * The sanitizer token.
+ */
+export const ISanitizer = new Token<ISanitizer>(
+  '@jupyterlab/apputils:ISanitizer'
+);
+/* tslint:enable */
```
Comment (jasongrout) [inline packages/apputils/src/tokens.ts:148]:
```suggestion
```

**Review by jasongrout — APPROVED** [review]
Looks and works great for me. Thanks! I've put in one suggestion for removing a tslint:disable comment.

**Comment by jasongrout** [comment]
Thanks!

**Pull request merged**